### ISSUE TITLE
.deb package

### ISSUE THREAD

**Issue by rllola** [HelloZeroNet/ZeroNet-linux#1]
Could it be interesting to have a .deb package ?

**Comment by theAkito** [comment]
Definitely, but using Python it makes everything worse. So it is not as easy, as just going by the book.

**Comment by rllola** [comment]
I have been experiencing with pyqtdeploy, pyinstaller and created a browser with ZeroNet embeded in it.

I can give a try for ZeroNet and see if it satisfies everyone.

**Comment by HelloZeroNet** [comment]
I think for .deb package you don't need pyinstaller as you can add the python3 package (and other libs) as dependency.
I'm not sure if all requirements available in the Debian apt repository and not sure what's the proper way to handle if that's the case.

btw here is the script that I used to generate the ZeroBundle for linux: https://github.com/HelloZeroNet/ZeroBundle/blob/py3/compile_linux.sh

It has evrything packed (openssl, zlib, python etc.) does not have any external dependency and I was able to successfully run it even on an old Debian 7, but I don't think that it would be accepted this way to any repo.

**Comment by rllola** [comment]
> I think for .deb package you don't need pyinstaller as you can add the python3 package (and other libs) as dependency.

True. It is not needed. I will try without those tools first and will post feedback.

I think first we can create a debian package. It would make the install a little bit more straightforward and maybe we can see if we can have it on the debian repo.
